tests/accessibility/getAccessibilityTree.test.ts

```diff
@@ -139,6 +139,76 @@ test(
   }),
 );
 
+test(
+  'role="presentation"',
+  withBrowser(async ({ utils, page }) => {
+    const body = (await page.$('body'))!;
+    await utils.injectHTML(`<h1 role="presentation">Sample Content</h1>`);
+    // Role="presentation" and role="none" are equivalent
+    // They make it as if the outer element wasn't there.
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(
+      `text "Sample Content"`,
+    );
```

Comment:
These comments are very helpful, thank you, @calebeby! 🙂 

Does it make sense to add a link to the [documentation you found](https://www.digitala11y.com/presentation-role/) here also?

Reply:
Or maybe above it?

```js
// See: https://www.digitala11y.com/presentation-role/
test('role="presentation"', withBrowser() => {});
```